geth-utils/gethutil/mpt/witness/gen_witness_transactions_test.go

```diff
@@ -42,15 +85,22 @@ func TestTransactions(t *testing.T) {
 }
 
 // No update for each step, just final proof.
+// This test uses hashed trie - no children are available in the stacktrie, we need
+// to get the by querying the database using hash values as keys.
 func TestGetProof(t *testing.T) {
```

Comment:
It looks like this test is more about the stacktrie. Can we rename it to reflect the property we want to check?